docs/experiment/guides/debugging-sample-ratio-mismatch.md

```diff
@@ -0,0 +1,73 @@
+<p></p>
+#### 3. Is the analysis time window the whole time the experiment was receiving traffic?
+<p style="display:inline;">The reason this is important is because if one variant causes users to return to the product more, then you may see more users than expected in that variant. For example, the experiment ran from January 1 to January 30, and the experiment was analyzed from January 15 to January 30. Say there are 100 users in control and treatment respectively every two weeks. Say the treatment you are experimenting on is so good that it makes all the users return the product every day and the control is so bad that they never come back. Then, during the analysis time window you will have 100 exposed users in control and 200 exposed users in treatment. Here the issue is that you are not doing an apples-to-apples comparison between control and treatment. If the analysis time window and the time the experiment was receiving traffic are different, change the analysis window to be the whole time the experiment was receiving traffic and see if there is still a SRM.</p>
+<p></p>
+#### 4. How much variant jumping (same user seeing both treatment and control) was there? Is it more than 5%? 
+<p style="display:inline;">You can find this graph in the Monitor tab of an experiment. The issue with variant jumping is that you do not know which variant to attribute the metric to. If a users sees control, then treatment, and then does the conversion event, did the user do the conversion event because of the treatment or did the user just need time to think about if they wanted to buy the item and the treatment had no effect on their probability of purchase? If there is variant jumping, is it because of anonymous users (people constantly logging in and out) or changing device ids? You can see this by looking at the <a href="https://help.amplitude.com/hc/en-us/articles/229313067-Look-up-event-data-for-individual-users">User Stream</a>.</p>
+<p></p>
+#### 5. Do you see a big difference in the conversion percentage between variants for the Assignment event -> Exposure event funnel?
+<p style="display:inline;">You can find this graph in the Monitor tab of an experiment. Plug in the conversion rates and sample size in <a href="https://www.socscistatistics.com/tests/ztest/default2.aspx">here</a> and see if its statistically significant or not at the 95% significance level. The conversion rates between variants should be similar (within randomness) because the assignment event should randomly split users into two equal cohorts. Since exposure event gets sent right before the user experiences a variant, there should be no difference between users since they have the same user experience from the time the assignment event is sent ot the time the exposure event is sent.</p>
```

Comment:
"You can find the Assignment to Exposure funnel chart in the Monitor tab of an experiment."